### PR TITLE
Remove hardcoded Pagerank dtype

### LINKED ISSUE #1737: [BUG] Pagerank errors on float64 weights
**Describe the bug**
Pagerank throws a misleading exn about requiring positive weights when weights are a `float64` series (`df.assign(ew=1.0)`), and the error goes away when cast to `float32`

It's fine if `float32` is required, but should improve the error message to a dtype error

**Steps/Code to reproduce bug**

```python
import cudf, cugraph, pandas as pd
fb_df = (
    pd.read_csv(
        'https://raw.githubusercontent.com/graphistry/pygraphistry/master/demos/data/facebook_combined.txt',
        sep=' ')
    .rename(columns={'0': 's', '1': 'd'})
)
fb_df['ew'] = 1.0
fb_gdf = cudf.DataFrame(fb_df)
G = cugraph.Graph()
G.from_cudf_edgelist(
            fb_gdf,
            source='s',
            destination='d',
            edge_attr='ew',
            renumber=True)
pagerank_df = cugraph.pagerank(G)
```
=>

```
cuGraph failure at file=../src/experimental/pagerank.cu line=110: Invalid input argument: input graph should have postive edge weights.
```

**Expected behavior**
Pagerank computed or a clear error about requiring `float32` dtype

**Environment overview (please complete the following information)**

conda 0.19 / docker / pascal

### PR DESCRIPTION
**Pull request by jnke2016**
fix hard coded pagerank dtype 
closes #1737

### CODE REVIEW

**Review by rlratzel — APPROVED** [review]

File: python/cugraph/link_analysis/pagerank_wrapper.pyx [hunk 1]
```diff
@@ -46,7 +46,7 @@ def pagerank(input_graph, alpha=0.85, personalization=None, max_iter=100, tol=1.
 
     df = cudf.DataFrame()
     df['vertex'] = cudf.Series(np.arange(num_verts, dtype=np.int32))
```
Comment (seunghwak) [inline python/cugraph/link_analysis/pagerank_wrapper.pyx:48]: Won't this crash with 64 bit vertex IDs as well? (and see the line 37, too).

Comment (jnke2016) [reply python/cugraph/link_analysis/pagerank_wrapper.pyx:48]: I talked to @ChuckHastings and he said some of legacy algorithm only support int32 vertex ID but those will be removed as we migrate to primitive based implementations

Comment (seunghwak) [reply python/cugraph/link_analysis/pagerank_wrapper.pyx:48]: I see, but isn't this code for PageRank? @ChuckHastings Do we still have legacy implementation for PageRank? AFAIK, we're using the primitive based implementation for both SG & MG PageRank.

Comment (jnke2016) [reply python/cugraph/link_analysis/pagerank_wrapper.pyx:56]: If Pagerank is primitive based, can I just do this ?
`[src, dst] = graph_primtypes_wrapper.datatype_cast([input_graph.edgelist.edgelist_df['src'], input_graph.edgelist.edgelist_df['dst']], [np.int32, np.int64])`

`df['vertex'] = cudf.Series(np.arange(num_verts, dtype=src.dtype))`

Comment (ChuckHastings) [reply python/cugraph/link_analysis/pagerank_wrapper.pyx:56]: Correct.  The legacy pagerank implementation was deleted a while back.  @jnke2016, your cast as above should work, I believe.

**Review by ChuckHastings — APPROVED** [review]
That looks better

**Review by seunghwak — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**